### PR TITLE
 Created table Group

### PR DESCRIPTION
**Pull request by SamantaTarun**
Create a new table named Group, with the following columns

GroupId
OwnerApplicationUserId
Name
Description
TopicTag
IOriginatorInfo interface columns.

please let me know if any modifications required.
#42

### CODE REVIEW

File: src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/Group.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE [dbo].[Group]
+(
+    [GroupId] BIGINT NOT NULL CONSTRAINT PK_Group PRIMARY KEY IDENTITY,
+    [OwnerApplicationUserId]  BIGINT NOT NULL,
```
Comment (efonsecab) [inline src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/Group.sql:4]: @tarunsamanta2k20 please add a foreign key to [ApplicationUser].[ApplicationUserId]

File: src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/Group.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    [Name] NVARCHAR(50) NOT NULL,
```
Comment (efonsecab) [inline src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/Group.sql:5]: @tarunsamanta2k20 please add a unique index for the name column

Comment (tarunsamanta2k20) [reply src/FairPlaySocialSln/FairPlaySocial.Database/dbo/Tables/Group.sql:5]: Done

**Review by efonsecab — APPROVED** [review]
@tarunsamanta2k20 
Approved. Thanks for the contribution.

**Comment by efonsecab** [comment]
Closes #42

**Comment by SamantaTarun** [comment]
@efonsecab please add me as a member of your organization. I wanna contribute more in tte future.

**Pull request merged**